api/src/index.ts

```diff
@@ -1,13 +1,24 @@
+require("dotenv").config();
 import Fastify from "fastify";
 import fetch from "node-fetch";
 import { generateNbTxsPerDay } from "./tasks/generateNbTxsPerDay";
 import { generateUniqueAddressGrowingPerDay } from "./tasks/generateUniqueAddressGrowingPerDay";
 import { generateTxsFeePerDay } from "./tasks/generateTxsFeePerDay";
 import { readData, writeData, FileData } from "./utils";
+import { tweetStats } from "../src/twitterBot/bullQueue";
 
 let cacheData: FileData | false = false;
 async function generateDataStats() {
   const currentData = readData();
+  if (currentData) {
+    await tweetStats
+      .add(
+        "tweet-stats",
+        { currentData: currentData },
+        { repeat: { cron: "* * * * *" } }
```

Comment:
@pradel I am facing a problem with the `repeat` part. 

it is adding jobs to queue in some strange way and also not executing them, as you can see here the end time is somewhere very far in the future <img width="365" alt="Screenshot 2021-08-12 at 13 47 32" src="https://user-images.githubusercontent.com/25903618/129185247-4d5c98a4-ca78-4944-87b8-563eedbcef45.png">

If I remove the repeat part it works smoothly and tweets immediately. 

I also tried to add intervals, but the same thing happens with that approach too.